src/pragmas.jl

```diff
@@ -95,12 +94,12 @@ function Quasar.visit_pragma(v, program_expr)
 end
 
 function parse_matrix(tokens::Vector{Tuple{Int64, Int32, Quasar.Token}}, stack, start, qasm)
-    inner = Quasar.extract_braced_block(tokens, stack, start, qasm)
+    inner = Quasar.extract_expression(tokens, Quasar.lbracket, Quasar.rbracket, stack, start, qasm)
```

Comment:
Why the switch from the more specific to the more general function? Are the delimiter-specific methods going away in the new Quasar.jl?

Reply:
Because `extract_braced_block` and other delimiter specific methods no longer exist in Quasar 0.0.2